x/bankplus/keeper/keeper.go

```diff
@@ -0,0 +1,131 @@
+package keeper
+
+import (
+	"github.com/line/lbm-sdk/codec"
+	sdk "github.com/line/lbm-sdk/types"
+	sdkerrors "github.com/line/lbm-sdk/types/errors"
+	bankkeeper "github.com/line/lbm-sdk/x/bank/keeper"
+	"github.com/line/lbm-sdk/x/bank/types"
+	paramtypes "github.com/line/lbm-sdk/x/params/types"
+)
+
+var _ Keeper = (*BaseKeeper)(nil)
+
+type Keeper interface {
+	bankkeeper.Keeper
+
+	AddBlockedAddr(ctx sdk.Context, address sdk.AccAddress)
+	DeleteBlockedAddr(ctx sdk.Context, address sdk.AccAddress)
+	IsBlockedAddr(address sdk.AccAddress) bool
+
+	InitializeBankPlus(ctx sdk.Context)
+}
+
+type BaseKeeper struct {
+	bankkeeper.BaseKeeper
+
+	ak           types.AccountKeeper
+	cdc          codec.BinaryMarshaler
+	storeKey     sdk.StoreKey
+	blockedAddrs map[string]bool
+}
+
+func NewBaseKeeper(
+	cdc codec.BinaryMarshaler, storeKey sdk.StoreKey, ak types.AccountKeeper, paramSpace *paramtypes.Subspace,
+	blockedAddrs map[string]bool,
```

Comment:
Inherited `BaseSendKeeper`'s `blockedAddrs` the name are duplicated, so I think It would be nice to change the name or add a comment description about why do the two coexist.

Reply:
The naming is very difficult to me. 😅
Can you recommend one?

Reply:
Oh!. What about `inactiveAddr`?